packages/TelegramClient-Tests.package/TCTAcceptanceTests.class/instance/testingMessageFrom..st

```diff
@@ -0,0 +1,23 @@
+accessing
+testingMessageFrom: aChatId
+	
+	^ TCCEvent newFromTDLibEvent: (Dictionary newFrom: {
+		'@type' -> 'updateNewMessage'.
+		'message' -> (Dictionary newFrom: {
+			'@type' -> 'message'.
+			'id' -> 9447669760.
+			'chat_id' -> aChatId.
+			'sender' -> (Dictionary newFrom: {
+				'@type' -> 'messageSenderUser'.
+				'user_id' -> 1673511834
+				}).
+			'content' -> (Dictionary newFrom: {
+				'@type' -> 'messageText'.
+				'text' -> (Dictionary newFrom: {
+					'text' -> 'Hallo'
+					})
+				}).
+				'is_outgoing' -> false.
+				'date' -> 1621267079
+			})
+		})
```

Comment:
`mockMessageFrom:` could be a better name. Same with `mockUserWith:`

Reply:
fixed in 1ddace8